helm-chart/csi-driver/templates/csi-linode-controller.yaml

```diff
@@ -108,6 +108,8 @@ spec:
               value: {{ .Values.enableMetrics | quote}}
             - name: METRICS_PORT
               value: {{ .Values.metricsPort | quote}}
+            - name: VOLUME_ENCRYPTION
```

Comment:
is there a reason we need this set as an env variable if it can be enabled per storage class?

Reply:
I am setting this env variable to use it for decision-making in controllerserver.go and controllerserver_helper.go . Basically this variable helps in the createVolume and attachVolume steps.

Reply:
we should be using `linodebs.csi.linode.com/encrypted: "true"` to decide if a volume is being encrypted instead of this environment variable

Reply:
Sure, I've updated the implementation to use this